modules/multi-architecture-import-imagestreams.adoc

```diff
@@ -9,16 +9,10 @@
 
 On an {product-title} {product-version} cluster with multi-architecture compute machines, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
 
-[IMPORTANT]
+[NOTE]
 ====
-The `referencePolicy.type` field of your `ImageStream` object must be set to the `Source` type for this procedure to run successfully.
-[source,yaml]
-----
-referencePolicy:
-    type: Source 
-----
+When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning pods to nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
```

Comment:
now that i think about it...i'm not sure this note belongs here -  it belongs somewhere in like a best practices section as it is generic and does not relate to imagestreams alone.

Reply:
Hello! I agree as well! I'm think I found a good place to move this section, but now I'm getting some build errors. So I'll get back to you once I figure out this issue. Thank you!